src/resources/companyResources.ts

```diff
@@ -0,0 +1,254 @@
+import { ResourceIF } from '@/interfaces'
+
+export const CompanyResources: Array<ResourceIF> = [
+  {
+    entityType: 'BEN',
```

Comment:
Should use an enum :)

Also maybe we can fetch the displayname below from shared corp-type-module?

Reply:
Absolutely!